zhmcclient/_adapter.py

```diff
@@ -347,3 +347,44 @@ def __repr__(self):
                 _ports=repr_manager(self._ports, indent=2),
             ))
         return ret
+
+    @logged_api_call
+    def change_crypto_type(self, crypto_type, zeroize=None):
+        """
+        Reconfigures a cryptographic adapter to a different crypto type.
+        This operation is only supported for cryptographic adapters.
+
+        Authorization requirements:
+
+        * Object-access permission to this Partition.
+        * Task permission to the "Partition Details" task.
```

Comment:
The auth reqs are incorrect. Needed is object-access permission to this Adapter, and task permission to the "Adapter Details" task.